scripts/releaseChangelog.mjs

```diff
@@ -246,7 +246,7 @@ Same changes as in \`@mui/x-data-grid-pro@__VERSION__\`${
     dataGridPremiumCommits.length > 0 ? ', plus:\n' : '.'
   }
 ${logChangelogSection(dataGridPremiumCommits)}${dataGridPremiumCommits.length > 0 ? '\n' : ''}
-### Date Pickers
+### Date and Time Pickers
```

Comment:
We try to use the same wording everywhere, I'm surprised that this occurrence was not spotted before

Reply:
🙈